tests/structure/data/1l2y.gro

```diff
@@ -304,4 +304,4 @@ GROtesk MACabre and Sinister
    20SER    HB2  302   0.221   1.134   0.027
    20SER    HB3  303   0.164   1.196   0.182
    20SER     HG  304   0.283   1.004   0.268
-   0.00000   0.00000   0.00000
+   2.00000   1.88562   1.63299   0.00000   0.00000   0.66667   0.00000  -0.66667   0.94281
```

Comment:
I did build a box arround the original 1l2y to be able to unit test the parsing.